internal/provider/resource_consul_cluster.go

```diff
@@ -583,30 +582,59 @@ func resourceConsulClusterUpdate(ctx context.Context, d *schema.ResourceData, me
 		return diag.Errorf("unable to fetch Consul cluster (%s): %v", clusterID, err)
 	}
 
-	// Fetch available upgrade versions
-	upgradeVersions, err := clients.ListConsulUpgradeVersions(ctx, client, cluster.Location, clusterID)
-	if err != nil {
-		return diag.Errorf("unable to list Consul upgrade versions (%s): %v", clusterID, err)
+	// Confirm update fields have been changed
+	sizeChanged := d.HasChange("size")
+	versionChanged := d.HasChange("min_consul_version")
+
+	if !sizeChanged && !versionChanged {
+		return diag.Errorf("at least one of: [min_consul_version, size] is required in order to upgrade the cluster")
```

Comment:
when is this error returned? When a config which does not have `ForceNew` enabled and nor is supported for update, is changed?